### PR TITLE
create character card in card.js and export

### PR DESCRIPTION
**Pull request by TmWgnr**

### CODE REVIEW

**Review by flexii07 — APPROVED** [review]
LGTM!

File: components/card/card.js [hunk 1]
```diff
@@ -1 +1,29 @@
...
+export function createCharacterCard(name, image, status, type, count) {
+  const cardListItem = document.createElement("li");
+  cardContainer.append(cardListItem);
+  cardListItem.classList.add("card");
+  cardListItem.innerHTML = `
+  <div class="card__image-container">
+    <img
+      class="card__image"
+      src="${image}"
+      alt="name"
```
Comment (dimitriosxmi) [inline components/card/card.js:12]: consider also using the `${name}` for the alt img attribute

**Review by dimitriosxmi — APPROVED** [review]
## LGTM les gooo

**Review by dimitriosxmi — APPROVED** [review]
## LGTM les gooo

**Pull request merged**